src/AccessibilityInsights.Core/Fingerprint/OutputFileLocation.cs

```diff
@@ -82,6 +83,8 @@ public bool Open()
                 {
                     throw;
                 }
+
+                e.ReportException();
```

Comment:
Is there a particular reason to only report the exceptions that weren't filtered out above?

Reply:
We only want to report each Exception once, and only when it gets eaten. If we were to call ReportException on the Exceptions that are being rethrown, they would be multiply-counted.